### PR TITLE
Task is ready to be checked

### PR DESCRIPTION
**Pull request by hubertkozik**
Task _1-points-joinby_ from _highcharts-maps_ is ready to be checked.

### CODE REVIEW

File: highcharts-api/highcharts-maps/1-points-joinby/main.js [hunk 1]
```diff
@@ -1,14 +1,29 @@
...
-        joinBy: ['iso-a3']
-    }]
-
-});
\ No newline at end of file
+    Highcharts.mapChart('container', {
+        chart: {
+            map: topology,
+            zoomType: 'xy',
+            height: 700
```
Comment (raf18seb) [inline highcharts-api/highcharts-maps/1-points-joinby/main.js:10]: These options are not necessary, you can remove them

File: highcharts-api/highcharts-maps/1-points-joinby/main.js [hunk 2]
```diff
@@ -1,14 +1,29 @@
...
-            ['AUS', 1]
-        ],
-        joinBy: ['iso-a3']
-    }]
-
-});
\ No newline at end of file
+    Highcharts.mapChart('container', {
+        chart: {
+            map: topology,
```
Comment (raf18seb) [inline highcharts-api/highcharts-maps/1-points-joinby/main.js:8]: You can shorten this by moving `topology` to `series.mapData` instead of `chart.map`.

**Review by raf18seb — APPROVED** [review]
Looks good, thanks! Can be shortened, but look good as it is now ;)

**Comment by hubertkozik** [comment]
Done.

**Pull request closed without merging**